test/cypress/support/commands.js

```diff
@@ -55,6 +55,33 @@ Cypress.Commands.add('menuGo', {}, (name) => {
   return cy.contains('#page-sidebar a', last).click({ force: true });
 });
 
+Cypress.Commands.add('apiLogin', {}, (username, password) => {
+  let loginUrl = '/api/automation-hub/_ui/v1/auth/login/';
```

Comment:
```suggestion
  let loginUrl = Cypress.env('prefix') + '_ui/v1/auth/login/';
```

(also configurable, depends on how the server is run)

Reply:
Ok.